helm-util/helm.libsonnet

```diff
@@ -97,4 +97,21 @@ local d = import 'github.com/sh0rez/docsonnet/doc-util/main.libsonnet';
         },
       }
     ),
+
+  '#setNamespace':: d.fn(
```

Comment:
it doesn't check if a resource is cluster wide (non namespaced). though adding a namespace to non-namespaced resource (eg. ClusterRole) can be created as well.